libs/remix-core-plugin/src/lib/compiler-artefacts.ts

```diff
@@ -80,12 +80,12 @@ export class CompilerArtefacts extends Plugin {
         const artefactsFilePaths = fileList.filter(filePath => {
           const filenameArr = filePath.split('/')
           const filename = filenameArr[filenameArr.length - 1]
-          if (filename === `${contractName}.json` || filename === `${contractName}_metadata.json`) return true
+          if (filename === `${contractName}_fullOP.json`) return true
```

Comment:
this can be discarded right?

Reply:
yes, this is still to update once we are done with the file name and content stuff

Reply:
this is updated.